internal/broadcasts/attendants.go

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: MIT
```

Comment:
could use a package level comment? looks like broadcasts doesn't have one. maybe that's not something you feel is sensible tho for packages under `internal/*`?

my feeling starting to read the file tho is like "okay so.. what is this? :D"